src/client/ui/dialogs/UpdateRequiredDialog.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styles from "./dialog.scss";
+import Button from "../Button";
+import Header from "../Header";
+
+export default class UpdateRequiredDialog extends Component {
+  static propTypes = {
+    downloadUrl: PropTypes.string
+  };
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div className={styles.dialogContainer}>
+        <Header title="Update Required" />
+        <div className={styles.updateRequiredContainer}>
+          This version of Spoke is no longer supported.<br />
```

Comment:
I think these forced updates will be more about bugs/security, so we can probably drop this sentence